apollo-router/tests/integration/response_cache.rs

```diff
@@ -461,6 +461,7 @@ async fn cache_control_merging_single_fetch() {
 }
 
 #[tokio::test]
+#[ignore = "flaky"]
```

Comment:
If you haven't already, I would ask in `#proj-router` if this change in ok. I assume it is, but best to check with the code owners.

Reply:
For apollo-federation work, I just skip building/checking `apollo-router`. Here's my quick check script:
```
        cargo build -p apollo-federation
        cargo build -p apollo-federation-cli
        cargo test -p apollo-federation --all-features
        cargo test -p apollo-federation-cli
        cargo clippy -p apollo-federation ${CLIPPY_OPTS}
        cargo clippy -p apollo-federation-cli ${CLIPPY_OPTS}
        cargo fmt -p apollo-federation -- --check ${FMT_CONFIGS}
        cargo fmt -p apollo-federation-cli -- --check ${FMT_CONFIGS}
```